src/main/resources/userEmailTemplate-feedbackSessionUnpublished.html

```diff
@@ -0,0 +1,16 @@
+Hello ${userName},
+
+<p/>
+${instructorFragment}
+
+<p/>
+The feedback responses for the following feedback session is now closed for viewing.
+<br>&nbsp;&nbsp; Course: [${courseId}]${courseName}
+<br>&nbsp;&nbsp; Feedback Session Name: ${feedbackSessionName}
+<br>
+<p/>
+If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
+<p/>
+Regards,
+<br>
+TEAMMATES Team.
```

Comment:
@damithc about the 3rd point, can't instructor intentionally unpublish the session. Should we need to mention 3rd point ?

Is it better ? 
```
Hello ${userName},
<p/>
${instructorFragment}

<p/>
The feedback responses for the following feedback session is unpublished and not available for viewing. 
<br>&nbsp;&nbsp; Course: [${courseId}]${courseName}
<br>&nbsp;&nbsp; Feedback Session Name: ${feedbackSessionName}
<br>
The response viewing link for this session, sent in the previous Email is closed and willn't work. You will be notified whenever the session will be published again.
<p/>
If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
<p/>
Regards,
<br>
TEAMMATES Team.
```